sdk/core/azure-core/CMakeLists.txt

```
@@ -125,7 +123,7 @@ else()
 endif()
 
 if(BUILD_TRANSPORT_CURL)
-  target_link_libraries(azure-core PRIVATE CURL::libcurl)
+  target_link_libraries(azure-core PUBLIC CURL::libcurl)
 endif()
 if(BUILD_TRANSPORT_WINHTTP)
   target_link_libraries(azure-core PRIVATE winhttp)
```

Make this public too.
```suggestion
  target_link_libraries(azure-core PUBLIC winhttp)
```

---

Per our offline conversation, we are deferring this decision till later.